npm run start fails with error message

The following error is displayed:

```bash 
ERROR in ./src/components/update-service/update-service.controller.ts
(246,63): error TS2345: Argument of type 'string[][]' is not assignable to parameter of type 'Many<PropertyName>'.
  Type 'string[][]' is not assignable to type 'PropertyName[]'.
    Type 'string[]' is not assignable to type 'PropertyName'.
      Type 'string[]' is not assignable to type 'symbol'.

ERROR in ./src/components/update-service/update-service.controller.ts
(247,45): error TS2345: Argument of type 'string[][]' is not assignable to parameter of type 'Many<PropertyName>'.
  Type 'string[][]' is not assignable to type 'PropertyName[]'.
```

I haven't tried to run the catalog standalone in a good while, so I'm not sure if this is a legitimate error or if standalone serve has fallen by the wayside (ie, works fine if running with `bower link` to console).  

@jeff-phillips-18 @spadgett @jwforres

--- a/app/app.ts
+++ b/app/app.ts
@@ -67,7 +67,7 @@ angular
   .component('logoutpage', logoutPage)
   .component('navigation', navigation)
   .constant('amTimeAgoConfig', {titleFormat: 'LLL'})
-  .run(function(IS_IOS) {
+  .run(function(IS_IOS: boolean) {
     if (IS_IOS) {
       // Add a class for iOS devices. This lets us disable some hover effects
       // since iOS will treat the first tap as a hover if it changes the DOM

--- a/app/mockServices/mockBinding.service.ts
+++ b/app/mockServices/mockBinding.service.ts
@@ -30,7 +30,7 @@ export class BindingService implements IBindingService {
   }
 
   public makeParametersSecret(secretName: any, parameters: any, owner: any) {
-    var secret = {
+    var secret: any = {
       apiVersion: 'v1',
       kind: 'Secret',
       metadata: {
@@ -45,10 +45,11 @@ export class BindingService implements IBindingService {
         }]
       },
       type: 'Opaque',
-      stringData: {}
+      stringData: {
+        parameters: JSON.stringify(parameters)
+      }
     };
 
-    secret.stringData['parameters'] = JSON.stringify(parameters);
     return secret;
   }
 

--- a/src/components/update-service/update-service.controller.ts
+++ b/src/components/update-service/update-service.controller.ts
@@ -238,7 +238,7 @@ export class UpdateServiceController implements angular.IController {
 
     _.each(this.secrets, (secret: any) => {
       var originalSecretParameters: any = JSON.parse(this.SecretsService.decodeSecretData(secret.data).parameters);
-      var originalSecretKeys = _.map(originalSecretParameters, (value: any, key: string) => {
+      var originalSecretKeys: any = _.map(originalSecretParameters, (value: any, key: string) => {
         // Make sure properties with a '.' in the name aren't treated as paths.
         return [key];
       });

